cdk/infrastructure/config.py

```diff
@@ -62,7 +62,7 @@
                         'construct_id': 'Postgres',
                         'on': True,
                         'props': {
-                            'snapshot_source_db_identifier': DEV_DATABASE_IDENTIFIER,
+                            'snapshot_source_db_identifier': PROD_DATABASE_IDENTIFIER,
```

Comment:
why did snapshot identifier change from DEV_DATABASE_IDENTIFIER to PROD_DATABASE_IDENTIFIER?   just want to make sure it's your intent to make this change.